.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+name: GitHub Tag Update
+
+on:
+  release:
+    types: [ published, edited ]
+
+permissions:
+  contents: read
+
+jobs:
+  actions-tagger:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+
+    steps:
+      - name: Repository checkout
+        uses: actions/checkout@v3
+
+      - name: Update tag
+        uses: Actions-R-Us/actions-tagger@v2
+        with:
+          publish_latest_tag: false
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I don't know... this force pushes the tags, right?  We don't yet automatically build the container image, though :-/

Reply:
Yes. When you push a new minor tag, e.g. `v1.0.1`, it will update the `v1` tag to point to the same commit.

Also, I can add the workflow for automatic container build.

Reply:
Ok, merging.